### PR TITLE
Move from Dict->Keyword. Update Docs

### PR DESCRIPTION
**Pull request by dbernheisel**
- Looks like the `Dict` module is deprecating. https://hexdocs.pm/elixir/Dict.html Changed usage from `Dict` to `Keyword`
- Updated the Readme to link to hex docs (solves #11)
- Updated the `def input` `@doc` to include another example that passes `class` to the input field.

### CODE REVIEW

File: lib/formulator.ex [hunk 1]
```diff
@@ -80,8 +84,8 @@ defmodule Formulator do
     input_type = options[:as] || :text
     input_class = options[:class] || ""
     options = options
```
Comment (drapergeek) [inline lib/formulator.ex]: While we're here, how about updating this follow our new style?
```
options =
  options
  |> Keyword.delete(:as)
```

File: mix.lock [hunk 2]
```diff
@@ -1,5 +1,5 @@
-%{"earmark": {:hex, :earmark, "1.0.1", "2c2cd903bfdc3de3f189bd9a8d4569a075b88a8981ded9a0d95672f6e2b63141", [:mix], []},
-  "ex_doc": {:hex, :ex_doc, "0.13.0", "aa2f8fe4c6136a2f7cfc0a7e06805f82530e91df00e2bff4b4362002b43ada65", [:mix], [{:earmark, "~> 1.0", [hex: :earmark, optional: false]}]},
-  "gettext": {:hex, :gettext, "0.11.0", "80c1dd42d270482418fa158ec5ba073d2980e3718bacad86f3d4ad71d5667679", [:mix], []},
-  "phoenix_html": {:hex, :phoenix_html, "2.6.2", "944a5e581b0d899e4f4c838a69503ebd05300fe35ba228a74439e6253e10e0c0", [:mix], [{:plug, "~> 1.0", [hex: :plug, optional: false]}]},
-  "plug": {:hex, :plug, "1.1.6", "8927e4028433fcb859e000b9389ee9c37c80eb28378eeeea31b0273350bf668b", [:mix], [{:cowboy, "~> 1.0", [hex: :cowboy, optional: true]}]}}
+%{"earmark": {:hex, :earmark, "1.0.1", "2c2cd903bfdc3de3f189bd9a8d4569a075b88a8981ded9a0d95672f6e2b63141", [:mix], [], "hexpm"},
```
Comment (drapergeek) [inline mix.lock]: What's up with this change?

Comment (dbernheisel) [inline mix.lock]: not sure how to prove it at this point, but I bet it's a difference in hex versions. Later version of hex is now including the repo information in the lockfile. https://github.com/hexpm/hex/blob/969926571818c9178ecb6a5d5566a42730aef5c2/CHANGELOG.md

File: lib/formulator.ex (hunk 1 above)
Comment (dbernheisel) [inline lib/formulator.ex]: 👍

File: lib/formulator.ex [hunk 3]
```diff
@@ -37,13 +41,13 @@ defmodule Formulator do
 
     case label_options do
       false -> input_without_label(form, field, options)
-      _ -> input_with_label(form, field, label_options, options)
+      _     -> input_with_label(form, field, label_options, options)
```
Comment (drapergeek) [inline lib/formulator.ex]: This is not part of our style guide. We don't line up operators.

File: mix.lock (hunk 2 above)
Comment (drapergeek) [inline mix.lock]: This should be a separate commit just to keep things clean.

File: lib/formulator.ex (hunk 3 above)
Comment (dbernheisel) [inline lib/formulator.ex]: 👍

**Comment by drapergeek** [comment]
:shipit:
